CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.0.1 - 2021-03-25
+
+### `Added`
+
+### `Changed`
+- [#40](https://github.com/qbicsoftware/variantstore-service/issues/40) Extended usage documentation and added examples
+- Prevent the creation of duplicate gene entries in the database
+
+### `Fixed`
+- [#42](https://github.com/qbicsoftware/variantstore-service/issues/42) - Fix parsing of Ensembl version
```

Comment:
Nice Changelog styling! 👍 
However i'd recommend linking the Pull requests instead of the issues to make tracking the changes easier

Reply:
Maybe I can even add both in case of fixes at least 👍

Reply:
Good Idea ➕